docs/source/en/model_doc/vits.md

```diff
@@ -93,21 +93,8 @@ from transformers import VitsTokenizer
 tokenizer = VitsTokenizer.from_pretrained("facebook/mms-tts-eng")
 print(tokenizer.is_uroman)
 ```
-
-If required, you should apply the uroman package to your text inputs **prior** to passing them to the `VitsTokenizer`, 
-since currently the tokenizer does not support performing the pre-processing itself.  
-
-To do this, first clone the uroman repository to your local machine and set the bash variable `UROMAN` to the local path:
-
-```bash
-git clone https://github.com/isi-nlp/uroman.git
-cd uroman
-export UROMAN=$(pwd)
-```
-
-You can then pre-process the text input using the following code snippet. You can either rely on using the bash variable 
-`UROMAN` to point to the uroman repository, or you can pass the uroman directory as an argument to the `uromaize` function:
-
+If the is_uroman attribute is True, the tokenizer will automatically apply the uroman package to your text inputs. 
```

Comment:
Can we encourage users to install `uroman` here and explain why it's required?

```sh
pip install --upgrade uroman
```

Reply:
added changes for docs.